spec/features/editing_images/upload_image_spec.rb

```diff
@@ -49,12 +49,11 @@ def and_i_upload_a_new_image
   def and_i_crop_the_image
     stub_publishing_api_put_content(@edition.content_id, {})
     click_on "Continue"
-    reset_executed_requests!
```

Comment:
Why has this been removed / why was it there in the first place?

Reply:
So this was because the upload process involves multiple preview generation requests. We were trying to check the last one of them, so we `reset_executed_requests!`, which was easier than trying to specify which request number we wanted to check. Does that help?

Reply:
yep, thanks :)